graphbolt/src/cuda/common.h

```diff
@@ -85,6 +88,48 @@ inline bool is_zero<dim3>(dim3 size) {
     }                                                                 \
   }
 
+#define GRAPHBOLT_DISPATCH_ELEMENT_SIZES(element_size, name, ...)             \
+  [&] {                                                                       \
+    switch (element_size) {                                                   \
+      case 1: {                                                               \
+        using element_size_t = uint8_t;                                       \
+        [[maybe_unused]] constexpr auto element_size_v =                      \
```

Comment:
`element_size_v` is known at compile time and using it instead of `element_size` which is known at runtime will lead to more optimized code in the future. However, we can do as you say and remove it, and when we need it, add it back.